README.rdoc

```diff
@@ -0,0 +1,28 @@
+== README
```

Comment:
Our projects seem to do markdown more than rdoc for the main readme. Should we do that here? Is there a benefit to rdoc?  I'm not familiar enough with Ruby idioms to know the tradeoffs.